internal/jobservice/repository/sqlite.go

```diff
@@ -0,0 +1,377 @@
+//go:generate moq -out sql_job_service_moq.go . JobTableUpdater
+package repository
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync"
+	"time"
+
+	_ "modernc.org/sqlite"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/armadaproject/armada/internal/jobservice/configuration"
+	js "github.com/armadaproject/armada/pkg/api/jobservice"
+)
+
+// JSRepoSQLite for persisting to DB.
+type JSRepoSQLite struct {
```

Comment:
Somehow "JS" prefix reads as JavaScript to me -- but I suppose I can get used to it

Reply:
Yes, I felt the same way when I first wrote it!  I thought about `JobSvcRepoSQLite` or even the more cumbersome `JobServiceRepoSQLite`, but both seemed just too long. I'd welcome any suggestions for concise alternatives.